draft-dejong-remotestorage-06.txt

```diff
@@ -261,12 +262,13 @@ Internet-Draft              remoteStorage                  November 2015
     Note that the use of strong ETags prohibits changing the response
     body based on request headers; in particular, the server will not be
     able to serve the same document uncompressed to some clients and
-    gzipped when requested by the client, since the two bodies would not
-    be identical byte-for-byte.
+    compressed to other clients when requested, since the two bodies 
+    would not be identical byte-for-byte.
 
     Servers MAY support Content-Range headers [RANGE] on GET requests,
-    but whether or not they do SHOULD be announced through the <ranges>
-    variable mentioned below in section 10.
+    but whether or not they do SHOULD be announced both through the 
+    <ranges> variable mentioned below in section 10 and the HTTP 
+    'Accept-Ranges' response header.
```

Comment:
And it doesn't seem that the HTTP spec prescribes any kind of *workflow* for the client to figure out if range requests are supported.

And if it's not required to send a "preflight" request to check whether range requests are supported, why should I do that extra roundtrip? See also http://stackoverflow.com/a/18745164